arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -0,0 +1,1041 @@
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Get cluster  successfully",
+            "schema": {
+              "$ref": "#/definitions/ClusterListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/clusters": {
+      "get": {
+        "tags": [
+          "ClusterListOperation"
+        ],
+        "operationId": "Clusters_List",
+        "description": "List cluster resource",
+        "x-ms-examples": {
+          "List clusters": {
+            "$ref": "../examples/ClusterListOperation_example.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Get cluster  successfully",
+            "schema": {
+              "$ref": "#/definitions/ClusterListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.ServiceFabric/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Lists all of the available ServiceFabric REST API operations.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CertificateDescription": {
+      "description": "Certificate description",
+      "properties": {
+        "thumbprint": {
+          "type": "string"
+        },
+        "thumbprintSecondary": {
+          "type": "string"
+        },
+        "x509StoreName": {
+          "type": "string",
+          "enum": [
+            "AddressBook",
+            "AuthRoot",
+            "CertificateAuthority",
+            "Disallowed",
+            "My",
+            "Root",
+            "TrustedPeople",
+            "TrustedPublisher"
+          ]
+        }
+      }
+    },
+    "SettingsParameterDescription": {
+      "description": "Settings parameter description",
```

Comment:
Better description - "Represents a setting entry in settings collection applied to fabric cluster "